wagtailnhsukfrontend/templates/wagtailnhsukfrontend/details.html

```diff
@@ -0,0 +1,10 @@
+<details class="nhsuk-details{{expander}}">
```

Comment:
Use a bool rather than passing a class to the template.
`<details class="nhsuk-details {% if expander %}nhsuk-expander{% endif %}">`

Reply:
Done.